sdk/batch/tests.data.yml

```diff
@@ -0,0 +1,11 @@
+trigger: none
```

Comment:
I suspect you should just remove this file as you will need more than this to get a live test pipeline up and running.